bcdaworker/worker/worker_test.go

```diff
@@ -124,112 +125,163 @@ func TestWorkerTestSuite(t *testing.T) {
 	suite.Run(t, new(WorkerTestSuite))
 }
 
-func (s *WorkerTestSuite) TestWriteResourceToFile() {
+func (s *WorkerTestSuite) TestWriteEOBToFile() {
```

Comment:
TestWriteResourceToFile has been broken up from its table test format because the test setup + mocking calls require jobArgs to have the resource types before the tests run, so that they can be used in the mock calls. This resulted in a sort of "chicken and egg" test setup. 

The difference between the old test and the new, is that it only tests one bene and it's no longer table driven.